conformance/README.md

```diff
@@ -77,27 +79,17 @@ make install-nkg-local-build
 ```
 
 #### *Option 2* Install Nginx Kubernetes Gateway from local already built image to configured kind cluster
+You can optionally skip the actual *build* step.
 
 ```makefile
 make install-nkg-local-no-build
 ```
 
-**Note:** You can optionally skip the actual *build* step. However, if choosing
-this option, the following step *must* be completed manually *before* the build step:
-
-- Set NKG_PREFIX=<nkg_repo_name> NKG_TAG=<nkg_image_tag> to preferred values.
-- Navigate to `deploy/manifests` and update values in `deployment.yaml` as specified in below code-block.
-- Save the changes.
-
- ```text
- .
- ..
- containers:
- - image: <nkg_repo_name>:<nkg_image_tag>
-   imagePullPolicy: Never
- ..
- .
- ```
+> Note:  If choosing this option, the following step *must* be completed manually *before* the build step:
```

Comment:
otherwise, it assumes we still need to run the build step. with the suggestion, it just assumes that the image was built some way
```suggestion
> Note:  If choosing this option, the following step *must* be completed manually *before* you build the image:
```